### ISSUE TITLE
Failed to convert from MySQL `YEAR` field type to PostgreSQL.

### ISSUE THREAD

**Issue by slotix** [slotix/dbconvert-streams-public#24]
## Steps to Reproduce

1. Define a table in MySQL with a `YEAR` field.
2. Use the provided scripts to create and fill the table in source mysql.

```sql
CREATE TABLE date_time_formats (
    id INT AUTO_INCREMENT PRIMARY KEY,
    date_column DATE,
    time_column TIME,
    timestamp_column TIMESTAMP,
    datetime_column DATETIME,
    year_created  year                         
);
```
```sql
INSERT INTO date_time_formats (date_column, time_column, timestamp_column, datetime_column, year_created)
VALUES
('2022-02-03', '12:34:56', '2022-02-03 12:34:56', '2022-02-03 12:34:56', '2022'),
('2021-12-15', '18:45:30', '2021-12-15 18:45:30', '2021-12-15 18:45:30', '2021'),
('2023-05-08', '08:15:00', '2023-05-08 08:15:00', '2023-05-08 08:15:00', '2023');
```

## Expected Behavior
The migration should successfully convert the MySQL `YEAR` field type to an equivalent type in PostgreSQL without errors.

## Actual Behavior
The migration fails with the following error:

```
Failed to create table {"error": "ERROR: type "year" does not exist (SQLSTATE 42704)", "DDL": {"CreateTable":"CREATE TABLE IF NOT EXISTS "public"."date_time_formats" ("id" SERIAL PRIMARY KEY NOT NULL, "date_column" DATE NULL, "time_column" TIME NULL, "timestamp_column" TIMESTAMP NULL, "datetime_column" TIMESTAMP NULL, "year_created" YEAR NULL); ","CreateIndexes":[]}}
[2024/02/05 22:44:12] [error] dbs-target-writer/main.go:232 [target] ERROR: type "year" does not exist (SQLSTATE 42704)
```

**Comment by slotix** [comment]
PostgreSQL doesn't have a built-in 'YEAR' data type. We can use 'INTEGER' or 'SMALLINT' instead. 

```sql
CREATE TABLE IF NOT EXISTS public.date_time_formats (
    id SERIAL PRIMARY KEY NOT NULL,
    date_column DATE NULL,
    time_column TIME NULL,
    timestamp_column TIMESTAMP NULL,
    datetime_column TIMESTAMP NULL,
    year_created INTEGER NULL
);
```

**Comment by slotix** [comment]
Fixed in the [version 0.10.3](https://stream.dbconvert.com/releases-2024#version-0-10-3)
